CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.10.1 - 2017-04-11
+
+- Update fury to 3.0.0-beta.1
```

Comment:
Any dependent that uses fury/fury-adapter-swagger would break unless they are pinned to a specific version.

If I have something like this as dependencies:

```json
{
  "dependencies": {
    "fury": "^2.1.0",
    "fury-adapter-swagger": "^0.10.0"
  }
}
```

Then when I install my dependencies I'd get a version of 2 fury and 0.10.1 of the adapter which is not compatible with fury2/minim. It's a breaking change. Other fury will provide older minim which would not pass elements down (ex: `resource.href.toValue()`).